bin/webpack/utils.js

```diff
@@ -0,0 +1,84 @@
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+/**
+ * Return plugin root path.
+ *
+ * @return {string} The plugin root path.
+ */
+const getPluginRootPath = () => {
+	return path.resolve( __dirname, '../../' );
+};
+
+/**
+ * Delete a file or directory.
+ *
+ * @param {string} _path The path to the file or directory.
+ *
+ * @return {void}
+ */
+const deleteFileOrDirectory = ( _path ) => {
+	if ( fs.existsSync( _path ) ) {
+		fs.rmSync( _path, { recursive: true } );
+	}
+};
+
+/**
+ * Determine the plugin version from the readme.txt file.
+ *
+ * @param {string} pluginPath The path to the plugin.
+ *
+ * @return {string|false} The plugin version or false if not found.
+ */
+const getPluginVersion = ( pluginPath ) => {
+	const readmePath = path.resolve( pluginPath, 'readme.txt' );
+
+	const fileContent = fs.readFileSync( readmePath, 'utf-8' );
```

Comment:
Not blocker: Better to use try catch for error handling.

```
let fileContent = '';
try {
	fileContent = fs.readFileSync( readmePath, 'utf-8' );
} catch ( error ) {
	log(
		formats.error(
			`Error reading the file "${ readmePath }": "${ error }"`
		)
	);
}
```

Reply:
There's no reason to handle the error here, in my opinion. We must end the operation here if it is unable to read or resolve the `readme.txt`. We don't need to add our error statements as NodeJS errors are already verbose.